account_commission_listing_fee/models/commission_settlement.py

```diff
@@ -49,6 +49,7 @@ def _prepare_listing_fee_product(self):
             "product_id": self.agent_id.listing_fee_product_id.id,
             "quantity": sum(settlement.number_of_items for settlement in self),
             "price_unit": self.agent_id.listing_fee_product_id.list_price * -1,
+            "tax_ids": self.agent_id.listing_fee_product_id.supplier_taxes_id,
```

Comment:
To be safe.

```suggestion
            "tax_ids": [(6, 0, self.agent_id.listing_fee_product_id.supplier_taxes_id)],
```

Reply:
`supplier_taxes_id` field is a many2many item. So ` [(6, 0, self.agent_id.listing_fee_product_id.supplier_taxes_id)]` will get an error.
<img width="507" alt="2023-06-27_11h47_15" src="https://github.com/qrtl/rmm-custom/assets/101688144/8e09dfa9-a63f-44c1-8bbe-892ccf6cdc8f">

Reply:
I missed to add `.ids` at the end.  Can you check again, please.

> `supplier_taxes_id field` is a many2many type.

This doesn't sound to be qualified as the reason of the error.

Reply:
Wasn't actually sure if the way you write would be supported.  I think we can also keep it as is if it works in case multiple taxes are involved.

Reply:
Both writing styles worked well with multiple taxes, but I updated `[6,0,ids]` writing style because it is used more in the Odoo code.